presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/ArrowFlightQueryRunner.java

```diff
@@ -115,8 +125,16 @@ public static void main(String[] args)
     {
         Logging.initialize();
 
+        int serverPort;
+        if (args.length >= 2) {
+            serverPort = Integer.valueOf(args[1]);
```

Comment:
Is this main function even used?

Reply:
I have only used it locally to run queries, which I thought is similar to [IcebergQueryRunner](https://github.com/prestodb/presto/blob/75dce5110fbc63e5167c9db89a1d64cf15556a55/presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergQueryRunner.java#L328) and others. Should I remove it?

Reply:
Yes better to remove

Reply:
I think the `main` functions in query runners serve a good purpose. It's useful for quick debugging when iterating on features for a particular connector. It is also a good starting point for new developers who want to contribute to a connector as the configuration (or lack thereof) makes it faster to spin up a presto instance than using the main `PrestoServer` class. Additionally, many query runners already have one of these methods: Hive, BlackHole, Iceberg, Druid, etc. Not all of them include it, but many do. I don't think it needs to be removed

Reply:
That being said, it might be prudent to split this into two commits. One for the open port fix, and one to add the main method

Reply:
Good point @ZacBlanco , I guess it's not really useful to pass in a port number as an arg or necessary to use a random port. I'll revert these changes.

Reply:
Okay if main() is useful